redbot/core/events.py

```diff
@@ -297,16 +297,22 @@ async def on_command_error(ctx, error, unhandled_by_cog=False):
                 msg = _("This command is on cooldown. Try again in 1 second.")
             await ctx.send(msg, delete_after=error.retry_after)
         elif isinstance(error, commands.MaxConcurrencyReached):
+            number = error.number
             if error.per is commands.BucketType.default:
                 msg = _(
                     "Too many people using this command."
                     " It can only be used {number} time(s) concurrently."
-                ).format(number=error.number)
+                ).format(number=number)
+            if error.per in (commands.BucketType.user, commands.BucketType.member):
+                msg = _(
+                    "You are using this command too many times."
+                    " It can only be used {number} time(s) per {type} concurrently."
```

Comment:
```py
"time(s)"
``` is not really translation friendly, that said it is already present in existing code.

```py
"You are using this command too many times."
``` feels odd. Its not that they are using it too many times, its that they are using it too many times before the previous complete, the sentence above could also apply to cooldowns etc.